src/jit/simd.cpp

```diff
@@ -225,7 +223,7 @@ var_types Compiler::getBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd,
         // Obtain base type by parsing fully qualified class name.
         //
         // TODO-Throughput: implement product shipping solution to query base type.
-        WCHAR className[256] = {'\\0'};
+        WCHAR className[256];
         WCHAR *pbuf = &className[0];
         int len = sizeof(className)/sizeof(className[0]);
         info.compCompHnd->appendClassName(&pbuf, &len, typeHnd, TRUE, FALSE, FALSE);   
```

Comment:
I got errors on that initializer, but have now determined that {0} works, so I've put the null-terminator back in.